presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/CommandExecutor.java

```diff
@@ -64,7 +63,7 @@ private int execute()
             try (InputStream processStream = process.getInputStream()) {
                 new Thread(() -> {
                     try {
-                        LOGGER.info("Output from command: {}\n{}", commandString, new String(ByteStreams.toByteArray(processStream)));
+                        LOGGER.info("Output from command: %s\n%s", commandString, new String(ByteStreams.toByteArray(processStream)));
                     }
                     catch (IOException e) {
                         LOGGER.error("Failed to log the process output");
```

Comment:
If we print stack trace once, then we should probably do it all the time. Add e to all error/warning logs.